tests/v2/actions/charts/charts.go

```diff
@@ -11,6 +11,7 @@ const (
 	serverURLSettingID = "server-url"
 	rancherChartsName  = "rancher-charts"
 	active             = "active"
+	metadataName       = "metadata.name="
```

Comment:
I don't see this used in this file. Can you move it to backuprestore.go instead?

Reply:
this was something @anupama2501 told me to update months ago when I first started working on this, no real relation to backup/restore as far as I'm aware

Reply:
the only place I saw it used is in backuprestore.go. it isn't used in the rest of the charts repo from what I could find.

Reply:
ah, upon further inspection I see where it can be updated in the other chart files and I recall now telling Anu that I only updated the BRO chart because it was in my scope, but if necessary I can update the rest of the files as well. I personally have no preference, I can update all of the hardcoded "metadata.name" in the charts files to use the new const, or I can remove the const an just leave it as as it is in the other chart files.